### PR TITLE
change labels of ZFIN and MGI genotypes to native syntax

### PR DESCRIPTION
**Pull request by justaddcoffee**
To address this ticket:
https://github.com/monarch-initiative/dipper/issues/596

### CODE REVIEW

File: dipper/sources/ZFIN.py [hunk 1]
```diff
@@ -543,15 +540,15 @@ def _process_genotype_features(self, limit=None):
 
         By the end of this method, we have built up the intrinsic genotype,
         with Monarch-style labels.
-        All ZFIN labels are added as synonyms (including the "sup" html tags).
+        Original ZFIN labels are added (including the "sup" html tags), and
```
Comment (kshefchek) [inline dipper/sources/ZFIN.py]: Do we want html tags in the primary label?  Ideally we would use [] or <> and add html tags on the client side.

Comment (kshefchek) [reply dipper/sources/ZFIN.py]: disregard this, it seems we end up with sup tags in our index either way, theres probably a larger problem to solve here,
https://beta.monarchinitiative.org/search/sup 
select genotype, then select Danio rerio

Comment (justaddcoffee) [reply dipper/sources/ZFIN.py]: yes, 16813 is lot of sup's...

Comment (justaddcoffee) [reply dipper/sources/ZFIN.py]: To clarify, at least some (most?) of these \<sup\>'s are coming from ZFIN themselves, as opposed to being added by Dipper. See for example fish_component.txt, second column, where some of our labels are coming from:
```
/raw/zfin $ more fish_components_fish.txt
ZDB-FISH-180403-6       li4Tg + MO1-apol1       ZDB-GENE-030131-3217    apol1   ZDB-MRPHLNO-150311-1    MO1-apol1                                       ZDB-GENO-180403-1       li4Tg
ZDB-FISH-170126-18      bmp15<sup>uc31/+</sup> ; uc44Tg                 ZDB-ALT-161121-9        uc44Tg  ZDB-TGCONSTRCT-161121-7 TgBAC(cyp19a1a:EGFP)                    ZDB-GENO-170126-15      bmp15<sup>uc31/+</sup> ; uc44Tg
ZDB-FISH-150901-26170   cacnb1<sup>mi90/mi90</sup>      ZDB-GENE-040718-399     cacnb1  ZDB-ALT-060313-2        mi90                                    ZDB-GENO-060313-1       cacnb1<sup>mi90/mi90</sup>
ZDB-FISH-150901-2194    dlc<sup>tit446/+</sup> + MO6-notch3     ZDB-GENE-000329-5       notch3  ZDB-MRPHLNO-070614-5    MO6-notch3                      ZDB-GENO-960809-7       AB      ZDB-GENO-100412-15      dlc<sup>tit446/+</sup>
ZDB-FISH-150901-21967   WT + MO1-crfb2 + MO3-crfb1      ZDB-GENE-070905-2       crfb2   ZDB-MRPHLNO-070906-4    MO1-crfb2                                       ZDB-GENO-030619-2       wild type (unspecified)
ZDB-FISH-181220-2       tbx2b<sup>p25bbtl/+</sup> ; ljr<sup>p23ahub/+</sup>     ZDB-RR-181218-1 ljr     ZDB-ALT-170303-2        p23ahub                                 ZDB-GENO-181220-2       tbx2b<sup>p25bbtl/+</sup> ; ljr<sup>p23ahub/+</sup>
```

**Comment by kshefchek** [comment]
+1 , this looks good to me

**Pull request merged**